### PR TITLE
Allow configuring multiple flags at a time

### PR DESCRIPTION
**Pull request by feedmeapples**
## What was changed

Allows to configure multiple params at a time:

```
$ tctl config set --namespace accounting --address 127.0.0.1:7233
namespace: accounting
address: 127.0.0.1:7233

$ tctl config get --namespace --address
namespace: accounting
address: 127.0.0.1:7233

$ cat ~/.config/temporalio/tctl.yaml
active: local
aliases: {}
environments:
  local:
    address: 127.0.0.1:7233
    namespace: accounting
version: next
```

## Why?

per comments in proposal https://github.com/temporalio/proposals/pull/39

## Checklist

1. Closes 

2. How was this tested:

3. Any docs updates needed?

### CODE REVIEW

File: cli/config.go [hunk 1]
```diff
@@ -66,50 +107,50 @@ func newConfigCommands() []*cli.Command {
...
-	}
-
-	key := c.Args().Get(0)
-	val := c.Args().Get(1)
-
-	if isRootKey(key) {
-		if err := tctlConfig.Set(c, key, val); err != nil {
-			return fmt.Errorf("unable to set property %s: %s", key, err)
+	for _, key := range keys {
+		if !c.IsSet(key) {
```
Comment (robholland) [inline cli/config.go:135]: This looks wrong, does that mean we can never set a key?

Comment (feedmeapples) [reply cli/config.go:135]: it sets a key if user provided it, 
for example for `tctl config set --namespace accounting` the code will find that `namespace` is present, however `address` is not present and will skip `address`

Comment (robholland) [reply cli/config.go:135]: Ah. Makes sense

**Review by robholland — APPROVED** [review]

**Comment by bergundy** [comment]
Using cli options (e.g. `--namespace`) seems like an unconventional choice here. Why was this chosen over a more familiar `key=value` syntax?

**Comment by bergundy** [comment]
@feedmeapples this doesn't implement the spec, in the spec there's no double dash for the options, I think the spec is a cleaner approach.

**Pull request merged**